apps/web/screens/Home.tsx

```diff
@@ -89,14 +92,13 @@ const Home: Screen<HomeProps> = ({
           />
         </Box>
         <Inline space={2}>
-          {featuredArticles.map(({ title, url }, index) => {
-            const isLatest = index + 1 === featuredArticles.length
+          {page.featuredThings.map(({ title, attention, thing }, index) => {
             return (
               <Tag
-                href={url}
-                key={url}
+                key={title}
+                href={makePath('article', thing.slug)}
```

Comment:
If these are always going to be article links then maybe it would be better wrap Tag with Link:

```js
      <Link as={makePath('article', thing.slug)} href={makePath('article', '[slug]')} passHref pureChildren>
        <Tag
          key={title}
          variant="darkerBlue"
          attention={attention}
        >
          {title}
        </Tag>
      </Link>
```